Tips-for-common-editors.md

```diff
@@ -0,0 +1,141 @@
+**A word of warning: Many code editors let you perform git actions from their graphical interfaces instead of through the command line. We strongly recommend that you use the command line instead. Many new contributors run into trouble because their editor's git interface is incorrectly configured.**
+
+## Table of Contents
+
+* [Visual Studio Code](#visual-studio-code)
+* [Sublime](#sublime)
+* [Atom](#atom)
+
+## Visual Studio Code
+
+If you are brand-new to Visual Studio Code, you can watch some [introductory videos](https://code.visualstudio.com/docs/getstarted/introvideos) to get started.
+
+To set up Oppia:
+
+1. Open the root `oppia/` folder in Code.
+
+2. You may be asked whether you trust the Oppia developers.
+
+   ![Trust modal shown by Code](images/commonEditors/vscodeTrust.png)
+
+   The code in the Oppia repository is safe to run, so you can click "yes." However, you can also edit code just fine if you click "no." You'll be in "restricted mode," which disables some features. If you ever find you need those features, you can change your trust settings later.
+
+3. You can navigate the files of the Oppia repository using the file tree on the left. Click on folders to show their contents, and click on files to open them in the editor that takes up the center of the window. On the far right is the minimap, which shows the file contents in small font so you can see more of the file at once.
+
+   ![Screenshot of the user interface with a file open for editing](images/commonEditors/vscodeEditor.png)
+
+4. To edit the file, just start typing!
+
+Tips:
+
+* Code has lots of extensions you can install. Here are some we find useful:
+
+  * `GitLens` gives you easy access to git information about your files. One particularly helpful feature shows the last commit to modify the line your cursor is on. If you hover over the annotation, GitLens shows more information about the commit.
+
+    ![GitLens annotation](images/commonEditors/vscodeGitLens.png)
+
+  * `Python` provides improved language support for Python. To configure the extension, we need to:
```

Comment:
You've just used "you" above and it changes to "we" here, which is weird.

Reply:
fixed